src/main/java/uk/gov/hmcts/probate/service/template/markdown/MarkdownDecoratorService.java

```diff
@@ -32,4 +36,11 @@ public String getPA17FormLabel(CaseData caseData) {
         }
         return "";
     }
+
+    public String getAuthenticatedTranslationLabel(CaseData caseData) {
+        if (authenticatedTranslationBusinessRule.isApplicable(caseData)) {
+            return "\n*" + AUTHENTICATED_TRANSLATION_WILL_TEXT;
```

Comment:
not sure if it makes a diff to markdown, but the other labels have some spaces after the star?

Reply:
It looks the same on test

Reply:
I can put in spaces though

Reply:
updated thanks